recipes/axiprop/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.5.2" %}
+
+package:
+  name: axiprop
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/hightower8083/axiprop/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 60c35fa42b5d887aad6c241fc22fcd57df94e8ef8912de0a2f57443a8292b43c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Reply:
Thanks! Waiting for resolution to https://github.com/conda-forge/staged-recipes/pull/24761#discussion_r1424676363 before applying :)

Reply:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```